test/edu-benefits/config/migrations.unit.spec.js

```diff
@@ -18,84 +18,84 @@ describe('EDU migrations', () => {
   describe('first migration', () => {
     it('should remove formId from return url', () => {
       const data1990 = {
-        formId: '1990',
+        formId: formConfig1990.formId,
```

Comment:
Oh interesting. Good idea. Was this causing a problem at any point?

Reply:
Yes, I had hard coded the url prefix rather than the complete form ID, which would've led to a different migration implementation.